sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -36,14 +36,17 @@ class EventHubConsumerClient(ClientBase):
     The main goal of `EventHubConsumerClient` is to receive events from all partitions of an EventHub with
     load-balancing and checkpointing.
 
-    When multiple `EventHubConsumerClient` operate within one or more processes or machines targeting the same
-    checkpointing location, they will balance automatically.
+    When multiple `EventHubConsumerClient` of the same event hub and same consumer group operate within one or more
+    processes or machines targeting the same checkpointing location, they will balance the number of partitions each
+    `EventHubConsumerClient` processes.
```

Comment:
The last `processes` can be removed.
In the last sentence, it is said `operate within one or more processes`, so in one python program(process), user can start multiple clients and they will balance.

` they will balance the number of partitions each EventHubConsumerClient.` I think is enough.

Reply:
I would go with something like:

When multiple `EventHubConsumerClients are running against the same event hub, consumer group and checkpointing location, the partitions will be evenly distributed among them.

Reply:
Changed to Anna' comment